README.md

```diff
@@ -1,21 +1,10 @@
 # Website Challenge
 
-## Overview
+Started with a bried chat to Amina about what to include. Then had a small chat about what design she's like where we ended up aiming for a slick design with Pastal/ Earthy Colours.
+https://coolors.co/1c0221-82ff9e-e6af2e-6b6d76-a2b9bc Currently using this colour system however will catch up and see if she likes it.
 
-For your first project on the bootcamp, we want you to make a website all about your pair partner using HTML and CSS.
+My first task was create the layout of the website. This took a little bit of time, ending up forcing each block into place using a larger block. Then I discovered flexbot. Never have I been so delighted and annoyed at the same time! It was wonderful, made instant improvements and was really easy! However it did make all the work I had put into my website completely redundant.
 
-The person you are making your website for is your partner, aka The Client. It is your job as a programmer to understand what the client wants from a profile site, assess what information is available and useful, design what serves the audience of users (aka the other bootcampers) best from the experience, and then create something which best solves those needs. You won't be given any guidance of what to build - you'll need to empathise with your users to see what they would find useful (or even survey and talk to them), and plan your site accordingly.
+Changed the Colour Scheme a bit as I wanted a darker background, then filled it with her info. Added some social media tags at the bottom.
 
-## Outcome
-
-- a single web page created with HTML and CSS
-- a profile describing your client
-- a short 1-page README.md file alongside it which describes how you went about the project (your thoughts, research, plans, and justification for decisions)
-
-## Extra
-
-This is not only a chance for you to practice the core skills of HTML and CSS, but also a chance to invest time in thinking about how to build a website for a client, and have a user-centric approach - a key learning objective of the course. You will need to actively communicate with the client to find out everything you need in order to design and build a website which represents them to the rest of the cohort.
-
-Although you will each have an individual repository in which to make the website, there is nothing stopping you pair programming with your partner on the project. Organise your time so that you can spend an equal amount of time on each project. For example, if you spend three hours coding together on Sunday, you would spend 1.5 hours pair programming on one site, before moving over to the other site.
-
-This project will span the first two weeks of the bootcamp, and you will be given time in live evening sessions to work on it as well.
+Struggled a little bit making the social media tags work. I wanted to use percentage so it fits better with other screens however ended up using pixels as the percentage struggled with flex.
```

Comment:
They're centered nicely, adding some extra padding or margin to the social media links could space them out more pleasingly. Alternatively, the flex property `flex-flow: space-around` migth produce the desired effect, and keep the page responsive.